plugin/al/translators/tests/testTranslators.py

```diff
@@ -60,6 +61,8 @@ def testMayaReference_RefLoading(self):
         def recordRefLoad(refNodeMobj, mFileObject, clientData):
             '''Record when a reference path is loading.'''
             path = mFileObject.resolvedFullName()
+            path = os.path.normpath(path)
+            path = os.path.normcase(path)            
             count = loadHistory.setdefault(path, 0)
```

Comment:
In a few of these tests we are using the name of the temp file we create as the key in a dictionary, and then try to use the path returned to us by Maya in a callback as a matching key.  We do not use these strings as actual file paths, just strings in the dictionary.  On Windows in particular this can mess up the tests because Maya might edit the string, in these cases it was mostly swapping / and \ characters.  
I thought the easiest way to fix this was to call both normpath and normcase, which should make both strings separator and case consistent.  Technically this might only be a problem right now on Windows but I have left it in for all OS’s since it works fine and it might help future tests.